### DISCUSSION TITLE
Unable to `docker login ghcr.io` 403

### DISCUSSION THREAD

**Discussion by simenbm** [community/community#27172] [Actions]
<p>Hi</p>
<p>When running <code>docker login ghcr.io</code> I get prompted for username and password. After entering my GitHub username and password I get an error stating:</p>
<pre><code>Error: error authenticating creds for "ghcr.io": Requesting bear token: invalid status code from registry 403 (Forbidden)
</code></pre>
<p>I tried with both my email and my username as the username. I’m using podman as a drop-in replacement for docker, but I am able to log in to <a href="http://docker.io" rel="noopener nofollow ugc">docker.io</a>. I have also been able to log in to <a href="http://ghcr.io" rel="noopener nofollow ugc">ghcr.io</a> last week, but now I am unable to.</p>
<p>Is it possible I’m doing something wrong on my end, or what could be the issue?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Try using a <a href="https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token">personal access token (PAT)</a> instead of the password. You need the <em>read:packages, write:packages</em> scopes to pull and push images, respectively.</p>

**Comment by rajahub108** [comment]
To login to ghcr.io: 
docker login ghcr.io -u username -p accesstokenclassic

Replace username with your github username, and password (GitHub personal access token) is obtained by going to : https://github.com/settings/tokens    

-> Generate New Token -> Classic

And selecting repos and packages permissions
